source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+   -  Faith’s phylogenetic Diversity
+   -  Pielou’s Evenness
+
+-  **Beta Diversity**
+
+   -  Jaccard distance
+   -  Bray Curtis distance
+   -  Unweighted UniFrac distance
+   -  Weighted UniFrac distance
+
+There is a very good discussion of diversity metrics and their meanings in a `qiime forum by Stephanie Orchanian`_.
+
+This method wraps several other methods, and it’s worthwhile to note that the steps in ``qiime diversity core-metrics-phylogenetic`` can be executed independently.
+
+One important consideration for diversity calculations is the Rarefaction depth. Above, we used alpha rarefaction and the sample summary to pick a rarefaction depth. So, for these analyses, we’ll use a depth of 1000 sequences per sample.
+
+.. note:: This step takes about 7 minutes
+
+.. command-block::
+
+   qiime diversity core-metrics-phylogenetic \
+     --i-table ./deblur_table.qza \
+     --i-phylogeny ./tree.qza \
+     --m-metadata-file ./metadata.tsv \
+     --p-sampling-depth 1000 \
+     --output-dir ./core-metrics-results
+
+Alpha Diversity
+---------------
+
+Alpha diversity asks whether the number of distribution of features within a sample differ between different conditions. The comparison makes no assumptions about the features that are shared between samples; two samples can have the same alpha diversity and not share any features. The rarified alpha diversity produced by ``q2-diversity`` is a univariate, continuous value and can be tested using common non-parametric statistical tests.
+
+Let’s test the relationship between the phylogenetic alpha diversity and evenness and our covariates of interest.
+
+.. command-block::
+
+   qiime diversity alpha-group-significance \
+     --i-alpha-diversity ./core-metrics-results/faith_pd_vector.qza \
+     --m-metadata-file ./metadata.tsv \
+     --o-visualization ./core-metrics-results/faiths_pd_statistics.qzv
+
+.. command-block::
+
+    qiime diversity alpha-group-significance \
+     --i-alpha-diversity ./core-metrics-results/evenness_vector.qza \
+     --m-metadata-file ./metadata.tsv \
+     --o-visualization ./core-metrics-results/evenness_statistics.qzv
+
+.. question::
+
+   For this exercise, we'll look at the group significance results for Faith's phylogenetic diversity and evenness.
+
+   Based on the group significance test, is there a difference in phylogenetic diversity by genotype? Is there a difference based on the donor?
+
+If we had a continuous covariate that we thought was associated with the alpha diversity, we could test that using ``qiime diversity alpha-correlation``. However, the only continuous variable in this dataset is the days since transplant.
+
+Beta Diversity
+--------------
+
+Next, we’ll compare the structure of the microbiome communities using beta diversity. Start by making a visualize inspection of the principle coordinates plots (PCoA) plots that were generated by emperor and ``core-metrics-results/weighted_unifrac_emperor.qzv`` into `view.qiime2.org`_
```

Comment:
```suggestion
Next, we’ll compare the structure of the microbiome communities using beta diversity. Start by making a visual inspection of the principle coordinates plots (PCoA) plots that were generated by emperor and ``core-metrics-results/weighted_unifrac_emperor.qzv``.
```

Just as a general note, q2view is not the only way to view results, I would prefer if we keep the "viewing" language agnostic, just to prevent confusion.

Reply:
Cool. I need to look into some of those.
And oh God, sorry about the proof reading stuff.